website/pages/en/features-timeline.js

```diff
@@ -0,0 +1,193 @@
+        |||   ```
+        ||| - [TypeScript 4.2](https://devblogs.microsoft.com/typescript/announcing-typescript-4-2/) support
+      </TimelineEvent>
+      <TimelineEvent name="Babel 7.12.0" date="Oct 2020">
+        ||| - Support for the [class static blocks](https://github.com/tc39/proposal-class-static-block) proposal
+        |||   ```js
+        |||   class A {"{"}
+        |||     static {"{"} initialize(A); {"}"}
+        |||   {"}"}
+        |||   ```
+        ||| - Support for imports and exports string names
+        |||   ```js
+        |||   let happy = "wooo!";
+        |||   export {"{"} happy as "😃" {"}"};
+        |||   ```
+        ||| - Parser support for the [Import Assertions](https://github.com/tc39/proposal-import-assertions) proposal
+        |||   ```js
+        |||   import json from "./foo.json" assert {"{"} type: "json" {"}"};
+        |||   ```
+        ||| - [TypeScript 4.1](https://devblogs.microsoft.com/typescript/announcing-typescript-4-1/) support
+      </TimelineEvent>
+      <TimelineEvent name="Babel 7.11.0" date="Jul 2020">
+        ||| - Enable the [Logical Assignment](https://github.com/tc39/proposal-logical-assignment/) and
+        |||   [Numeric Separator](https://github.com/tc39/proposal-numeric-separator) Stage 4 proposals by default
+        ||| - Parser support for the [Decimal](https://github.com/tc39/proposal-decimal) proposal
+        |||   ```js
+        |||   console.assert(0.1m + 0.2m === 0.3m);
+        |||   ```
+        ||| - [TypeScript 4.0](https://devblogs.microsoft.com/typescript/announcing-typescript-4-0/) support
+      </TimelineEvent>
+      <TimelineEvent
+        name="@babel/eslint-parser"
+        link="https://babeljs.io/blog/2020/07/13/the-state-of-babel-eslint"
+      />
+      <TimelineEvent name="Babel 7.10.0" date="May 2020">
+        ||| - Enable parser support for the [`import.meta`](https://github.com/tc39/proposal-import-meta/) Stage 4 proposal by default
+        ||| - Support for the [Ergonomic brand checks for Private Fields](https://github.com/tc39/proposal-private-fields-in-in) proposal
+        |||   ```js
+        |||   class Car {"{"}
+        |||     #plate;
+        |||     race(other) {"{"}
+        |||        if (#plate in other) console.log("Racing against another car!");
+        |||     {"}"}
+        |||   {"}"}
+        |||   ```
+      </TimelineEvent>
+      <TimelineEvent
+        name="babel-polyfills"
+        link="https://github.com/babel/babel-polyfills"
+      />
+      <TimelineEvent name="Babel 7.9.0" date="Mar 2020">
+        ||| - `bugfixes` option in `@babel/preset-env`, to workaround browsers bugs rather than compiling whole Features
+        ||| - [TypeScript 3.8](https://devblogs.microsoft.com/typescript/announcing-typescript-3-8/) support
+        ||| - Support for Flow `declare` class fields
+        ||| - Support for the [automatic](https://it.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html") JSX runtime
```

Comment:
```suggestion
        ||| - Support for the [automatic](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html) JSX runtime
```

I think we could add an example here too if we wanted